File: modul5/Python/PRAK502-2310817210029-Putra_Whyra_Pratama_Setiawan.py
import math

def mutlak(angka):
    if angka < 0:
        angka = angka * -1
    return angka

def hitung(nilai1, nilai2):
    hitung = nilai1 - nilai2
    if hitung < 0:
        hitung = hitung * -1
    return hitung

a, c, b, d = map(int, input().split())
Hasil = hitung(a, b) + hitung(c, d)
print(mutlak(Hasil))

File: modul5/C/PRAK502-2310817210029-Putra_Whyra_Pratama_Setiawan.c
#include <stdio.h>
#include <math.h>

int mutlak(int angka){
    if(angka < 0) {
        angka = angka*-1;
    }
    return angka;
}

int hitung(int nilai1, int nilai2){
    int hitung;
    hitung = nilai1 - nilai2;
    if(hitung < 0){
        hitung = hitung*-1;
    }
    return hitung;
}

int main(){
    int a, b, c, d;

    scanf("%d %d %d %d", &a, &c, &b, &d);

    int Hasil = hitung(a, b) + hitung(c, d);
    printf("%d",mutlak(Hasil));

    return 0;
}